.github/labeler.yml

```diff
@@ -37,6 +33,10 @@ topic-dask:
 topic-DataTree:
   - xarray/core/datatree*
 
+topic-documentation:
+  - doc/*
+  - doc/**/*
```

Comment:
Ah this needs to skip `whats-new` otherwise everything gets tagged with topic-documentation.